internal/scheduler/metrics/state_metrics.go

```diff
@@ -11,26 +13,35 @@ import (
 	"github.com/armadaproject/armada/pkg/armadaevents"
 )
 
+const noCheckpointLabelValue = "none"
```

Comment:
worth moving this into consts.go?

Reply:
Possibly, somewhat consts is shared constants but also I see your point.

I need to make a follow up PR so will fix it in that one.